ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/clientlibs/less/backtotop.less

```diff
@@ -0,0 +1,36 @@
+.bottom-to-top-container {
+    justify-content: center;
```

Comment:
@ReenaOmana : component is not coming in mobile

![image](https://user-images.githubusercontent.com/38690824/84896822-3f474200-b0c2-11ea-93dc-89a747f59c35.png)

Reply:
I'm able to see the component in mobile view Rishabh. Please see the screenshot
![mobileview](https://user-images.githubusercontent.com/65391905/84991119-b41e8880-b163-11ea-8fb9-eaa6666a7dc3.JPG)